plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -1567,6 +1567,305 @@ var _ = Describe("validator", func() {
 				})
 			})
 
+			Context("kubelet config checks", func() {
+				var (
+					worker          core.Worker
+					resourceCPU1    = resource.MustParse("2")
```

Comment:
I think `resourceCPU1` and `resourceCPU2` have specific semantics: One is "enough capacity", the other is "insufficient capacity" – can we reflect this in the names, so it becomes clear when reading the tests without looking up the specific values?
The same applies to the memory variables as well.

Reply:
Hmm.. but these are not capacity, like, I'm just using two values each to create more and less `kubeReserved` and `systemReserved` resources. And we have cases where they add up and exceed the capacity or exceed the capacity by themselves. I'm not sure renaming them to enough and insufficient will make things clear in this case.

Reply:
You're right, we're also having the scenarios where only the sum of the two reservations is too high, while both values alone are perfectly fine – I didn't think this through entirely.

My initial goal was to make the tests more readable without having to look up the individual values and compare them to the worker's capacity in the test – if we cannot have variable names that are helpful, maybe we should just inline the values, so the tests become more clear?

By now, I'm leaving this up to you: If you feel this doesn't add to the tests readability, it's fine to leave this as-is.

Reply:
Okay, now I checked the same, the CPU and Memory fields are taking pointers to `resource.Quantity`, so we would have to assign them to variables.

Reply:
> My initial goal was to make the tests more readable without having to look up the individual values and compare them to the worker's capacity in the test

Yes, got it. I guess we can leave it as it is, given the above reason and we have used simple values like 2, 3 etc. It should not be too hard to follow.